config/stack_jedi.yaml

```diff
@@ -0,0 +1,263 @@
+cmake:
```

Comment:
how is `stack_jedi` different that `stack_noaa`?
I understand a few (all) are `build: YES`, but we need that on NOAA too (all except wcoss).
So, rather than adding a `stack_jedi`, can we work with `stack_noaa`?
Thoughts?

Reply:
The main difference here from stack_noaa.yaml is building hdf5 and netcdf as shared libraries.  However, after re-checking the CMakeLists.txt file for the noaa-emc/gsi build, I'll try to add the directives FIND_HDF5_HL and FIND_HDF5 to see if that helps.  From memory, those directives were getting pre-empted by the cmake/Modules/platforms/Generic.cmake script in the master branch of the noaa-emc/gsi repo.